examples/analyse_output.py

```diff
@@ -1,12 +1,11 @@
-from blobmodel import Model
+from blobmodel import Model, Geometry
 import xarray as xr
 import matplotlib.pyplot as plt
 import os
 
 if not os.path.isfile("./example.nc"):
-    bm = Model(
-        Nx=100, Ny=50, Lx=10, Ly=10, dt=0.1, T=20, blob_shape="exp", num_blobs=1000
-    )
+    geo = Geometry(Nx=100, Ny=50, Lx=10, Ly=10, dt=0.1, T=20,)
+    bm = Model(blob_shape="exp", num_blobs=1000)
```

Comment:
Not sure if this is intentional, but in this way the default model initialization is slightly more complex than it was before. You can keep the same model constructor there was before, and initialize the Geometry in the model constructor (in such a way that the lines here would not need to be changed). Just resolve this thread if you did it in this way intentionally.

Reply:
I see that I forgot to pass on geo to Model, is it clear then?